boot_shutdown_interface/config/boot_shutdown.param.yaml

```diff
@@ -0,0 +1,12 @@
+/**:
+  ros__parameters:
+    startup_timeout: 180
+    preparation_timeout: 180
+    debug:
```

Comment:
This parameter for debug is not needed.

Reply:
@ito-san 
 I removed this parameter for debugging.
6516b0ef9b117751ab3df4840f489ee60d2d56df